docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -92,40 +94,41 @@ spec:
     ports:
       - containerPort: 443
 ```
-When a pod is attached to a certain network, it needs to set its "network" field:
+When a pod is attached to a certain network, it needs to set its "network" in labels:
 
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
   name: nginx
+  labels:
+    arktos.futurewei.com/network: vpc-1
 spec:
   containers:
   - name: nginx
     image: nginx
     ports:
       - containerPort: 443
-  network: vpc-1
 ```
 
 
-When a pod is attached to a certain network and it wants to specify subnet or IP, it needs to be specified in pod fields:
+When a pod is attached to a certain network and it wants to specify subnet or IP, it can be expressed in annotations:
 
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
   name: nginx
+  labels:
+    arktos.futurewei.com/network: vpc-1
+  annotations:
+    arktos.futurewei.com/nics: [{"name": "eth0", "ip": "192.168.0.12"}]
 spec:
   containers:
   - name: nginx
     image: nginx
     ports:
       - containerPort: 443
-  network: vpc-1
-  nics:
-    - subnet: subnet-1
-      ip: 192.168.0.12
```

Comment:
Since these spec is removed, Mizar needs to have a way to know the subnet this pod belongs to.

Reply:
```annotations:
￼     arktos.futurewei.com/nics: [{"name": "eth0", "subnet": "subnet-1", "ip":"192.168.0.12"}]```
like above example, user can provide subnet in nics annotation.

Reply:
So you will keep the NIC arguments unspecified?

Reply:
based on previous design, agreed-upon keys include name, subnet, ip, portid. I'll call it out explicitly in the updated design doc.